src/lib/constants.ts

```diff
@@ -1,2 +1,11 @@
 export const BUILD_SCRIPT_URL =
   "https://raw.githubusercontent.com/Microsoft/bedrock/master/gitops/azure-devops/build.sh";
+
+export const projectInitDependencyErrorMessage =
```

Comment:
Can these variable names be changed to match the all uppercase convention?

ie. PROJECT_INIT_DEPENDENCY_ERROR_MESSAGE.

or something similar?